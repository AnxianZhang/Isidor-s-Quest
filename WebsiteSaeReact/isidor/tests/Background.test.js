import renderer from 'react-test-renderer'
import BackgroundPicture from '../component/Background'

describe('<Backgroud>', () => {
    it('Has 1 child', () => {
        const tree= renderer.create(<BackgroundPicture />).toJSON()
        expect(tree.children.length).toBe(1)
    })

    it("renders correctly", ()=>{
        const tree = renderer.create(<BackgroundPicture/>).toJSON()
        expect(tree).toMatchSnapshot()
    })
})