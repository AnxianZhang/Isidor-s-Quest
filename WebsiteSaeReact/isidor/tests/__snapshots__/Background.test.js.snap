// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Backgroud> renders correctly 1`] = `
<View
  accessibilityIgnoresInvertColors={true}
>
  <Image
    style={
      [
        {
          "bottom": 0,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        },
        {
          "height": undefined,
          "width": undefined,
        },
        undefined,
      ]
    }
  />
</View>
`;
